src/realm/util/thread.cpp

```diff
@@ -114,6 +133,27 @@ bool Thread::get_name(std::string& name) noexcept
     name_2[max - 1] = '\0';              // Eliminate any risk of buffer overrun in strlen().
     name.assign(name_2, strlen(name_2)); // Throws
     return true;
+#elif defined(_WIN32)
+    // GetThreadDescription was only introduced in Windows 10 1607, so we need to dynamically load it.
+    HRESULT(WINAPI * GETTHREADDESCRIPTION)(HANDLE hThread, PWSTR * threadDescription) = nullptr;
+#if !defined(NTDDI_WIN10_RS1) || NTDDI_VERSION < NTDDI_WIN10_RS1
+    auto proc_address = GetProcAddress(GetModuleHandle(L"Kernel32.dll"), "GetThreadDescription");
```

Comment:
```suggestion
    static const auto proc_address = GetProcAddress(GetModuleHandle(L"Kernel32.dll"), "GetThreadDescription");
```

and ditto other comments from above.